Sources_v3/StreamChatUI/ChatChannel/MessageComposerCommandCellView.swift

```diff
@@ -0,0 +1,87 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+class MessageComposerCommandCellView: View {
+    // MARK: Properties
+
+    public lazy var commandImageView: UIImageView = UIImageView().withoutAutoresizingMaskConstraints
+    public lazy var commandNameLabel: UILabel = UILabel().withoutAutoresizingMaskConstraints
+    public lazy var commandNameSubtitleLabel: UILabel = UILabel().withoutAutoresizingMaskConstraints
+    private lazy var textStackView: UIStackView = UIStackView().withoutAutoresizingMaskConstraints
+
+    // MARK: - Appearance
+
+    override func setUp() {
+        super.setUp()
+        // TODO: Implement in `CIS-465 Implement SuggestionsViewController`
+    }
+
+    override func defaultAppearance() {
+        super.defaultAppearance()
+        commandNameLabel.font = UIFont.preferredFont(forTextStyle: .footnote).bold()
+
+        commandNameSubtitleLabel.font = .preferredFont(forTextStyle: .caption1)
+        commandNameSubtitleLabel.textColor = .systemGray
+
+        if #available(iOS 13.0, *) {
+            backgroundColor = .systemBackground
+            commandNameLabel.textColor = .label
+        } else {
+            backgroundColor = .white
+            commandNameLabel.textColor = .black
```

Comment:
Now we have `ColorPalette` in `UIConfig` so it's good idea to use it here for colors.

Reply:
Thanks, fixed